riscof-plugin/sail_cSim/README.md

```diff
@@ -0,0 +1,100 @@
+# Riscof Plugin for RISC-V SAIL C-Emulator
```

Comment:
Why is all this stuff in a sail_cSim subdirectory? It's already in the sail-riscv repo, so sail is implied. Most (all?) of this should be usable for both the C and OCaml emulators, so the cSim is unnecessary, but even if it weren't, at least use the same naming convention (c_emulator) as sail-riscv.

Reply:
There is a plugin for every model that riscof can run - that is every implementation by every vendor.
We wanted to ensure that if whoever changes a model keep the plugin from breaking, and keeping them in the same directory means they won't get out of sync. We don't want thousands of plugins (assuming they're all different) overloading the riscof repo with hundred or thousands of plugin PRs, so distributing them into the repo whose model is kept is much cleaner.

Reply:
Regarding Ocaml: unless and until we know for sure that the plugins are identical, we should probabaly keep it as-is.
If Ocaml is determined to be identical, AND we start using the Ocaml simulator for the arch rest flow, then we can think about a rename. Right now we have no plans to use the Ocaml.